src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -30,7 +33,45 @@ object ScalafixEnable {
     } yield p -> fullVersion
   }
 
-  lazy val command = Command.command(
+  lazy val command =
+    if (isSemanticdbPluginAvailable) withSemanticdbPluginCommand
+    else withoutSemanticdbPluginCommand
+
+  private lazy val withSemanticdbPluginCommand = Command.command(
```

Comment:
this may bring some regressions for sbt 1.3+ users, so it probably needs some manual testing (with scala steward maybe?)